app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/image_import_workflow.rb

```diff
@@ -0,0 +1,111 @@
+class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::ImageImportWorkflow < ManageIQ::Providers::AnsiblePlaybookWorkflow
+  def load_transitions
+    self.state ||= 'initialize'
+
+    {
+      :initializing      => {'initialize'         => 'waiting_to_start'},
+      :start             => {'waiting_to_start'   => 'pre_execute'},
+      :pre_execute       => {'pre_execute'        => 'execute'},
+      :execute           => {'execute'            => 'running'},
+      :poll_runner       => {'running'            => 'running'},
+      :post_execute      => {'running'            => 'post_execute_poll'},
+      :post_execute_poll => {'post_execute_poll'  => 'post_execute_poll'},
+      :finish            => {'*'                  => 'finished'},
+      :abort_job         => {'*'                  => 'aborting'},
+      :cancel            => {'*'                  => 'canceling'},
+      :error             => {'*'                  => '*'}
+    }
+  end
+
+  def post_execute
+    ems = ExtManagementSystem.find(options[:ems_id])
+
+    body = {
+      :source        => 'url',
+      :imageName     => options[:miq_img][:name],
+      :osType        => options[:miq_img][:os],
+      :diskType      => options[:diskType],
+      :imageFilename => "#{options[:session_id]}.ova",
+      **options[:cos_pvs_creds]
+    }
+
+    ems.with_provider_connection(:service => 'PCloudImagesApi') do |api|
+      response = api.pcloud_cloudinstances_images_post(ems.uid_ems, body, {})
+      context[:task_id] = response.taskref.task_id
+
+      started_on = Time.now.utc
+      update!(:context => context, :started_on => started_on)
+      miq_task.update!(:started_on => started_on)
+    end
+
+    route_signal(:post_execute_poll, 'importing image into PVS', 'running')
```

Comment:
You only need `route_signal` if you depend on filesystem state or otherwise require being on the same pod in podified.  It looks like `post_execute_poll` would be just fine with `queue_signal`

Reply:
done